airflow/api_connexion/openapi/v1.yaml

```diff
@@ -5193,6 +5194,11 @@ components:
           items:
             type: string
           description: The value can be repeated to retrieve multiple matching values (OR condition).
+        order_by:
+          type: string
+          description: |
+            The name of the field to order the results by. Prefix a field name
+            with `-` to reverse the sort order.
```

Comment:
I think this should also mention the version it was added.
```suggestion
            The name of the field to order the results by. Prefix a field name
            with `-` to reverse the sort order.

            *New in version 3.0.0*
```

Reply:
Updated docs, thanks!